src/types/dict.rs

```diff
@@ -46,6 +57,13 @@ pyobject_native_type_core!(
     #checkfunction=ffi::PyDictValues_Check
 );
 
+#[cfg(not(PyPy))]
+impl PyDictValues {
+    pub fn as_sequence(&self) -> &PySequence {
+        unsafe { PySequence::try_from_unchecked(self) }
+    }
+}
+
```

Comment:
This should have a comment similar to the existing `as_sequence` methods (same for PyDictItems)